lib/node_modules/@stdlib/random/array/tools/docs/types/index.d.ts

```diff
@@ -21,19 +21,105 @@
 /* eslint-disable max-lines */
 
 import binary = require( '@stdlib/random/array/tools/binary' );
+import binaryFactory = require( '@stdlib/random/array/tools/binary-factory' );
```

Comment:
@Planeshifter This is an instance where our tooling assumes that a package has TypeScript declarations, but we don't have them (yet) for these packages. It would be good to improve here.

Reply:
I don't believe that this should be blocking for merging this PR.